src/Runner.Common/Util/NodeUtil.cs

```diff
@@ -13,10 +13,21 @@ public static class NodeUtil
         public static readonly ReadOnlyCollection<string> BuiltInNodeVersions = new(new[] { "node12", "node16" });
 #endif
 
+        public static string GetNodeVersion()
+        {
+            return System.Environment.GetEnvironmentVariable(Constants.Variables.Agent.ForcedActionsNodeVersion);
```

Comment:
do we need a method that only has a single line?

Reply:
NIT: Let's remove the method and do it inline